dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 pfultz2/rocm-recipes
-ROCm-Developer-Tools/HIP@roc-3.0.0
+# ROCm-Developer-Tools/HIP@rocm-3.7.0
```

Comment:
This version of hip is only for hcc. When we switch to hip-clang, we can remove these dependencies.

Reply:
@pfultz2 As you can see, this PR in fact switches all tests (but static tests) to hip-clang. Unfortunately, docker build fails. I've tried some fixes, but no success yet. ___I will appreciate if you look at build logs at Jenkins and give me some advice.___

/cc @shurale-nkn

Reply:
Change reverted. This is used only in hip3.0 based dockers that will be used for static checks for a while. @shurale-nkn Is working on a patch that should switch static checks to hip-clang.